seatunnel-shade/seatunnel-hadoop3-3.1.4-uber/pom.xml

```diff
@@ -80,43 +80,21 @@
                             <relocations>
                                 <relocation>
                                     <pattern>org.apache.commons.io</pattern>
-                                    <shadedPattern>shade.org.apache.commons.io</shadedPattern>
+                                    <shadedPattern>${seatunnel.shade.package}.org.apache.commons.io</shadedPattern>
                                 </relocation>
                                 <relocation>
                                     <pattern>com.google.common.base</pattern>
-                                    <shadedPattern>${seatunnel.shade.package}.hadoop.com.google.common.base</shadedPattern>
+                                    <shadedPattern>${seatunnel.shade.package}.com.google.common.base</shadedPattern>
                                 </relocation>
                                 <relocation>
                                     <pattern>com.fasterxml.jackson</pattern>
-                                    <shadedPattern>${seatunnel.shade.package}.hadoop.com.fasterxml.jackson</shadedPattern>
+                                    <shadedPattern>${seatunnel.shade.package}.com.fasterxml.jackson</shadedPattern>
                                 </relocation>
                             </relocations>
                         </configuration>
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
```

Comment:
You shouldn't delete here, because `seatunnel-hadoop3-3.1.4-uber` is a module of seatunnel project. If you remove `build-helper-maven-plugin` the modules which dependency `seatunnel-hadoop3-3.1.4-uber` will use the hadoop unshaded jar.

Reply:
Done